language/documentation/book/src/creating-coins.md

```diff
@@ -183,27 +183,20 @@ We can convert our `Coin` into a resource just by adding the `resource` keyword
         }
 ```
 
-Let's publish our `Coin` module again and re-run our script:
+Let's clean the storage, publish our `Coin` module again and re-run our script:
 
 ```shell
+$ move clean
 $ move publish src/modules
 $ move run src/scripts/test-coin.move
 error:
 
-   ┌── scripts/test-coin.move:6:13 ───
-   │
+   ┌── src/scripts/test-coin.move:5:36 ───
+   │ 6 │         let _coin = Coin::mint(100);
+   │                                    ^ Invalid return   ·
```

Comment:
What's up with the `·` at the end?

And this is fine, but previously we had tried to just describe the kind of error without actually copy-pasting the error from the compiler. Sometimes those can be a bit verbose as to try to give all the information needed, which can be overkill while learning

Reply:
> What's up with the · at the end?

Apparently the new line was not properly copy-pasted. Should now be fixed.

I think that the previous version was also the raw output of an older version of the compiler, which was slightly more verbose.
The address of the Coin module in the error output was also inconsistent: 0x1 instead of 0x2.